mantidproject/mantid
# Add a Region Of Interest mode to the Instrument view

This adds a region of interest (ROI) mode to the instrument window.

This gathers together functionality from other modes, and add an intersection based view for applying ROIs.

This also changes the Mask/Group tab name to "draw" and the instrument tree tab name to "instrument".

fixes #11824
### Documentation

The page http://www.mantidproject.org/MantidPlot:_Instrument_View has been updated
### release notes

http://www.mantidproject.org/ReleaseNotes_3_5_UI_Changes#Instrument_View
### To Test
1. Load up some data
2. Play around with adding masks, ROI's and groups, check the interplay between them.
3. Test the options under the Save or apply and save button in each mode.
